libs/ngx-dsfr/alert/src/lib/testing/alert.component.spec.ts

```diff
@@ -115,9 +116,12 @@ describe('DsfrAlertComponent', () => {
 
     it('should have the right description', async () => {
       testHost.testDescription = testDescription;
-      fixture.detectChanges();
-      const description = await dsfrAlertHarness.getAlertDescription();
-      expect(description).toEqual(testDescription);
+      // Tip for testing component which implements AfterViewInit and ViewChild,
+      // without this, the constant 'description' will be undefined
+      setTimeout(async () => {
```

Comment:
I've tested it and on my end, there's no need for this...
You were right to remove `fixture.detectChanges();` but there's apparently no need to wait for an extra cycle of Change Detection, so you can unwrap lines 122 and 123 from within `setTimeout`